### PR TITLE
chore: update .gitignore

### PR DESCRIPTION
**Pull request by EdwinChang24**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,364 @@
...
+### Remix ###
+node_modules
 /.cache
+/build
+/public/build
+
+### Remix+CloudflarePages ###
 /functions/\[\[path\]\].js
 /functions/\[\[path\]\].js.map
-/functions/metafile.*
```
Comment (Sophon96) [inline .gitignore:6]: Can we add this back?

Comment (Sophon96) [reply .gitignore:6]: Is there a reason why this is omitted?

**Review by Sophon96 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (EdwinChang24) [reply .gitignore:6]: I'll add it back.

**Comment by Sophon96** [comment]
Can you also add `.dev.vars` back?

**Review by Sophon96 — APPROVED** [review]
LGTM!

**Pull request merged**